src/utils.js

```diff
@@ -126,6 +126,12 @@ export const getAWSEnvironment = () => {
 
 export const isAwsEnvironment = () => !!process.env['LAMBDA_RUNTIME_DIR'];
 
+export const isTimeoutTimerEnabled = () =>
+  !!(
+    process.env['LUMIGO_TIMEOUT_TIMER_ENABLED'] &&
```

Comment:
IMO it will be nice to extract `'LUMIGO_TIMEOUT_TIMER_ENABLED'` to a const because it is used in both the `get` and `set` functions.

Reply:
👍 
also did there some nice DRY Change take a look